### PR TITLE
fpga bitstream module: increase test coverage

### PR DESCRIPTION
**Pull request by bart0sh**
Increased coverage of the ls pkg/fpga/bitstream module to more than 90%

NOTE: I didn't touch aocx module as FPGA plugin doesn't support OpenCL anymore.

### CODE REVIEW

File: pkg/fpga/bitstream/gbs_test.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	}
+
+	_, err = gbs.RawBitstreamData()
+	if err != nil {
+		t.Errorf("unexpected data error: %+v", err)
+		return
+	}
+
+	intUUID := gbs.InterfaceUUID()
+	if interfaceUUID != interfaceUUID {
```
Comment (rojkov) [inline pkg/fpga/bitstream/gbs_test.go]: Isn't it a definition of negated identity?

File: pkg/fpga/bitstream/gbs_test.go [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+		t.Errorf("unexpected Interface UUID value: %s", intUUID)
+	}
+
+	installPath := gbs.InstallPath("")
+	if installPath != filepath.Join(interfaceUUID, typeUUID)+".gbs" {
+		t.Errorf("unexpected Install Path value: %s", installPath)
+	}
+
+	typUUID := gbs.AcceleratorTypeUUID()
+	if typeUUID != typeUUID {
```
Comment (rojkov) [inline pkg/fpga/bitstream/gbs_test.go]: and this?

Comment (bart0sh) [reply pkg/fpga/bitstream/gbs_test.go]: Thank you for the review. my bad, will fix right away.

**Comment by bart0sh** [comment]
@mythi @rojkov Does anything else need to be done for this PR? if not, please merge. Thanks.

**Comment by rojkov** [comment]
LGTM, but the Merge button is disabled for me because I'm not code owner.

**Comment by bart0sh** [comment]
@rojkov > LGTM, but the Merge button is disabled for me because I'm not code owner.

OK, thanks.

@kad can you review/merge this PR please? It seems that we're the only owners of this module.

**Pull request merged**